[2022-05-08 09:16:54,946] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag_v4.second_task manual__2022-05-08T09:16:52.490876+00:00 [queued]>
[2022-05-08 09:16:54,972] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag_v4.second_task manual__2022-05-08T09:16:52.490876+00:00 [queued]>
[2022-05-08 09:16:54,973] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-08 09:16:54,975] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-08 09:16:54,976] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-08 09:16:55,072] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2022-05-08 09:16:52.490876+00:00
[2022-05-08 09:16:55,089] {standard_task_runner.py:52} INFO - Started process 509 to run task
[2022-05-08 09:16:55,101] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag_v4', 'second_task', 'manual__2022-05-08T09:16:52.490876+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmphzk80ea4', '--error-file', '/tmp/tmp2i4d71r2']
[2022-05-08 09:16:55,120] {standard_task_runner.py:80} INFO - Job 11: Subtask second_task
[2022-05-08 09:16:55,332] {task_command.py:369} INFO - Running <TaskInstance: first_dag_v4.second_task manual__2022-05-08T09:16:52.490876+00:00 [running]> on host ef82a602b7b6
[2022-05-08 09:16:55,545] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=faceit
AIRFLOW_CTX_DAG_ID=first_dag_v4
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-08T09:16:52.490876+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-08T09:16:52.490876+00:00
[2022-05-08 09:16:55,551] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-08 09:16:55,553] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hello world task -2!']
[2022-05-08 09:16:55,582] {subprocess.py:85} INFO - Output:
[2022-05-08 09:16:55,592] {subprocess.py:92} INFO - hello world task -2!
[2022-05-08 09:16:55,594] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-08 09:16:55,693] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag_v4, task_id=second_task, execution_date=20220508T091652, start_date=20220508T091654, end_date=20220508T091655
[2022-05-08 09:16:55,823] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-08 09:16:55,957] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
